### PR TITLE
Small fix in Linux install instruction + minor CSS change

### PR DESCRIPTION
**Pull request by javl**
* Fixes a typo in the Linux instructions (`make Run` should be `make RunRelease`)
* Adds a small note on finding the amount of cores available for compiling
* Removes the hover effect from the method name headers in the documentation (see commit for more info)

### CODE REVIEW

**Comment by javl** [comment]
Sorry for putting these in the same pull request. I always forget committing in a branch with an open pull request will add the commit to the request. I can revoke the request and create two new ones if you prefer (though the two commits are very small so should be quite clear what they do).

File: files/setup/linux-install/index.html.mako [hunk 1]
```diff
@@ -40,7 +40,7 @@ If you want to install the project generator, a tool that allows to create and u
 
 ```
 cd OF/scripts/linux
-./compilePG.sh
+./compilePG.sh -j3
```
Comment (javl) [inline files/setup/linux-install/index.html.mako]: Adds `-j3` just like in the `./compileOf.sh -j3` example above.

File: themes/openframeworks/assets/css/style.css [hunk 2]
```diff
@@ -419,6 +419,7 @@ a:active {
 	text-decoration: none;
 }
 a.nohover:hover {
+	color: #EE3987;
```
Comment (javl) [inline themes/openframeworks/assets/css/style.css]: The only other element using `nohover` are some `img`s, so adding `color` does not influence those.

**Comment by arturoc** [comment]
thanks!

**Pull request merged**